Core/clim-basic/geometry/region-predicates.lisp

```diff
@@ -66,12 +77,52 @@
   (every (lambda (r) (region-contains-position-p r x y))
          (standard-region-set-regions self)))
 
-(defmethod region-contains-position-p ((self standard-region-difference) x y)
-  (and (region-contains-position-p (standard-region-difference-a self) x y)
-       (not (region-contains-position-p (standard-region-difference-b self) x y))))
+(defmethod region-contains-position-p ((region standard-region-difference) x y)
+  (let ((region-a (standard-region-difference-a region))
+        (region-b (standard-region-difference-b region)))
+    (and (region-contains-position-p region-a x y)
+         (not (region-contains-position-p region-b x y)))))
+
+(defmethod region-contains-position-p ((self point) x y)
+  (and (coordinate= (point-x self) x)
+       (coordinate= (point-y self) y)))
```

Comment:
Please use `point-position` -- `point-x` **and** `point-y` call it anyway which is crazy expensive.

Reply:
fixed